controllers/api_v1/keyword_test.go

```diff
@@ -105,6 +105,131 @@ func TestKeywordAPIControllerDbTestSuite(t *testing.T) {
 	suite.Run(t, new(KeywordAPIControllerDbTestSuite))
 }
 
+func (s *KeywordAPIControllerDbTestSuite) TestFetchKeywordsWithValidParams() {
+	keyword := models.Keyword{UserID: s.user.ID, Keyword: faker.Name()}
+	db.GetDB().Create(&keyword)
+
+	headers := http.Header{}
+	headers.Set("Authorization", "Bearer test-access")
+
+	resp := testHttp.PerformRequest(s.engine, "GET", "/api/v1/keywords", headers, nil)
+
+	assert.Equal(s.T(), http.StatusOK, resp.Code)
```

Comment:
Can we count the list keyword in the response data @gutakk as we don't know the response for this case, the test should make sure the response has `1 keyword`

Reply:
That's make sense. Added in cc78491f42a7970319663e132494785c6bd0a4ff